### PR TITLE
add speakers to website

### PR DESCRIPTION
**Pull request by OriolAbril**
As usual, the branch can be previewed in my [fork's gh-pages](https://oriolabril.github.io/pymcon/index.html). Pictures and bios are missing, please someone hijack the PR and add them, copy them here or share in slack.

### CODE REVIEW

**Comment by canyon289** [comment]
Let's get picture of each keynote and add it before merging.

**Comment by canyon289** [comment]
Made some edits and leaving up for review. If its good we should merge

File: _includes/2017_data/header.html [hunk 1]
```diff
@@ -22,8 +22,9 @@
                         <a href="./cfp">CFP</a>
                     </li>
                     <li>
-                        <!--This tag has to be speakers because it's the only way the
-                            css works for the planning committee thumbnails-->
+                        <a href="./#speakers">Speakers</a>
+                    <li>
```
Comment (OriolAbril) [inline _includes/2017_data/header.html]:
```suggestion
                    </li>
```

File: _includes/2017_data/header.html [hunk 2]
```diff
@@ -22,8 +22,9 @@
...
+                    <li>
                         <a href="./#organizers">Organizers</a>
                     <li>
```
Comment (OriolAbril) [inline _includes/2017_data/header.html:30]:
```suggestion
                    </li>
```

File: _includes/2017_data/header-home.html [hunk 3]
```diff
@@ -22,8 +22,8 @@
                         <a href="./cfp">CFP</a>
                     </li>
                     <li>
-                        <!--This tag has to be speakers because it's the only way the
-                            css works for the planning committee thumbnails-->
+                        <a class="page-scroll" href="#speakers">Speakers</a>
+                    <li>
```
Comment (OriolAbril) [inline _includes/2017_data/header-home.html:27]:
```suggestion
                    </li>
                    <li>
```

File: _includes/2017_data/header.html [hunk 4]
```diff
@@ -22,10 +22,11 @@
...
                     </li>
                     <li>
-                        <!--This tag has to be speakers because it's the only way the
-                            css works for the planning committee thumbnails-->
-                        <a href="./#organizers">Organizers</a>
+                        <a href="./#speakers">Speakers</a>
+                    </li>
                     <li>
+                        <a href="./#organizers">Organizers</a>
+                    </li>
```
Comment (OriolAbril) [inline _includes/2017_data/header.html:29]:
```suggestion
                    </li>
                    <li>
```

**Pull request merged**